remotespark/datawidgets/ipythondisplay.py

```diff
@@ -0,0 +1,11 @@
+from IPython.core.display import display, HTML
+
+
+class IpythonDisplay(object):
```

Comment:
This shouldn't be a class.  `display_to_ipython` and `html_to_ipython` should just be top-level functions in this module.